assume/scenario/loader_csv.py

```diff
@@ -372,7 +378,7 @@ def read_units(
         bidding_strategies = {
             key.split("bidding_")[1]: unit_params[key]
             for key in unit_params.keys()
-            if key.startswith("bidding_")
+            if key.startswith("bidding_") and unit_params[key] != 0
```

Comment:
I think you generally do not want to have empty values as a given strategy here - so None and empty string as well, so I would just check `and unit_params[key]`

```suggestion
            if key.startswith("bidding_") and unit_params[key]
```